Index: discord.py
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/discord.py b/chopper.py
rename from discord.py
rename to chopper.py
--- a/discord.py	(revision 18843abaab13a639462389f09501e4b432d11452)
+++ b/chopper.py	(date 1782016656165)
@@ -27,8 +27,8 @@
 from dataclasses import dataclass
 from typing import Optional
 
-import discord
-from discord.ui import Button, Modal, TextInput, View
+import chopper
+from chopper.ui import Button, Modal, TextInput, View
 from dotenv import load_dotenv
 
 
